holidayextras/brands
# Legoland forms

#### What does this PR do? (please provide any background)
all the forms on legoland are not aligned with LEGO branding guidelines

#### How can this be tested?
`cd pattern-library && npm link ../brands/legoland`
check http://localhost:8687/elements/forms/index.html and see that placeholders are grey and input text is black. Italics are removed

#### Any tech debt?

#### Screenshots / Screencast

#### What gif best describes how you feel about this work?
![legobatman](https://user-images.githubusercontent.com/3897844/36103546-87ea99de-1007-11e8-8678-beb428a664d2.gif)



---

**Reviewer**

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Ensured the release has the correct version bump.
- Checked all the work locally on pattern library.
